### ISSUE TITLE
Red/green "indicator labels"

### ISSUE THREAD

**Issue by will-h** [inex/IXP-Manager#745]
We use what i'll call 'indicator labels' with red/green to indicate statuses:

![image](https://user-images.githubusercontent.com/2152161/138851134-15cc077e-dbdb-4a29-bdf8-b01a85c808f5.png)

Personally I get confused, e.g. when i see a red ![image](https://user-images.githubusercontent.com/2152161/138851043-56515000-eb9e-4de5-8b14-f75a3ba3f9fb.png) i expect the peer to be a route-server client. 

Moreover it's not great to use red and green for this because of the prevalence of red/green colour blindness.

Is it feasible to add some symbols for instance: ✅ ❌ ⛔️  against these labels?

**Comment by ichilton** [comment]
To clarify this, IXP Manager always shows the "RS Client" indictor, regardless of the rs client status. It then uses red or green to indicate whether rs client is on or off.

However, unless you are specifically aware of this, when you go in to a port and see the presence of an RS Client indicator, your assumption is that RS Client is on - it's confusing.

**Comment by nickhilliard** [comment]
yeah, this is a problem which needs a bit of thought.  A naive grep also turns up:

```
resources/views/search/additional/ip.foil.php:28
resources/views/search/additional/ip.foil.php:38
resources/views/rs-prefixes/help.foil.php:34
resources/views/rs-prefixes/help.foil.php:79
resources/views/customer/overview-tabs/overview.foil.php:148
resources/views/customer/overview-tabs/overview.foil.php:153
resources/views/customer/overview-tabs/peers/table.foil.php:43
resources/views/peering-manager/tabs/table.foil.php:51
```

**Comment by barryo** [comment]
Generally ack'ing this but closing - this can/will all be addressed on the next UI/UX rework.
